repositories/user_repository.py

```
@@ -0,0 +1,64 @@
+import json
+import os
+from typing import Optional, List
```

Ruff (0.14.11)

`typing.List` is deprecated, use `list` instead

(UP035)